docs/graph_engine/temporal.rst

```diff
@@ -0,0 +1,119 @@
+Temporal graph support
+======================
+
+DeepGNN suports temporal graphs, i.e. graphs with timestamps assigned to each element:
+nodes, edges, or features. Lets explore how to use them.
+
+Node
+----
+
+A new node can be added or removed to a graph.
+A JSON schema for temporal nodes is an extension of `regular<data_spec.html>`_ nodes with two new fields: `created_at`
+and `removed_at` with values recorded in cardinal format. Here is a concrete example:
+
+.. code-block:: json
+
+	{
+	"node_id": 42,
+	"node_type": 0,
+	"node_weight": 1.0,
+	"binary_feature": {"0": "microsoft", "1": "xbox"},
+    "created_at": 13,
+    "removed_at": 23,
+	"edge":[
+		{
+		"src_id": 42,
+		"dst_id": 99,
+		"edge_type": 0,
+		"weight": 2.0,
+		},
+	],
+	}
+
+Here we describe a node with id 42, two binary features, a single outgoing edge and it existed from time 13 to time 23.
+If created_at field is not specified, it is assumed to be 0. If removed_at field is not specified, it is assumed to be
+DeepGNN doesn't support changing node type over time yet. A simple workaround is to delete
+the node and create a new node with a new type or store node type as a feature.
+
+Edge
+----
+
+A temporal edge is similar to node described above, it has two new fields `created_at` and `removed_at`.
+Unlike nodes, there might be multiple edges between two nodes. Here is an extension of the previous example:
```

Comment:
I think this can be written more concisely

Reply:
How can you make it shorter?

Reply:
"An edge can be added to or removed from the graph at specific times. Edges have two fields: `created_at`
and `removed_at` with values recorded in Unix time format. Here is an extension of the previous example:"